clients/src/main/java/org/apache/kafka/clients/producer/RecordMetadata.java

```diff
@@ -24,18 +24,20 @@
 public final class RecordMetadata {
```

Comment:
Also would be good to add a comment to RecordMetadata class description what timestamp is actually returned (either set by client, producer, or broker).